src/test/java/formflow/library/controllers/ShowErrorStackTraceTest.java

```diff
@@ -10,18 +10,18 @@
 @SpringBootTest(properties = {"form-flow.path=flows-config/test-flow.yaml", "form-flow.error.show-stack-trace=true", "form-flow.error.pretty-print-packages=formflow,org.springframework"}, webEnvironment = RANDOM_PORT)
 public class ShowErrorStackTraceTest extends AbstractBasePageTest {
 
-  @Test
-  void showErrorPageWithStackTraceWhenPropertyEnabled() {
-    navigateTo("flow/inputs/asdf");
-    String errorMessages = testPage.findElementsByClass("data-table").get(0).getText();
-    String stackTrace = testPage.findElementsByClass("stack-trace").get(0).getText();
-    assertThat(errorMessages).containsSequence("Timestamp:");
-    assertThat(errorMessages).containsSequence("Status: 500");
-    assertThat(errorMessages).containsSequence("Message: Could not find flow=inputs in templates");
-    assertThat(errorMessages).containsSequence("Path: /flow/inputs/asdf");
-    assertThat(errorMessages).containsSequence("Exception: java.util.NoSuchElementException");
-    assertThat(stackTrace).containsSequence("java.util.NoSuchElementException: Could not find flow=inputs in templates");
-  }
+//  @Test
+//  void showErrorPageWithStackTraceWhenPropertyEnabled() {
+//    navigateTo("flow/inputs/asdf");
+//    String errorMessages = testPage.findElementsByClass("data-table").get(0).getText();
+//    String stackTrace = testPage.findElementsByClass("stack-trace").get(0).getText();
+//    assertThat(errorMessages).containsSequence("Timestamp:");
+//    assertThat(errorMessages).containsSequence("Status: 500");
+//    assertThat(errorMessages).containsSequence("Message: Could not find flow=inputs in templates");
+//    assertThat(errorMessages).containsSequence("Path: /flow/inputs/asdf");
+//    assertThat(errorMessages).containsSequence("Exception: java.util.NoSuchElementException");
+//    assertThat(stackTrace).containsSequence("java.util.NoSuchElementException: Could not find flow=inputs in templates");
+//  }
```

Comment:
I'm assuming you meant to bring this back in?

Reply:
Good catch, thanks!